.github/workflows/web_builds.yml

```
@@ -40,7 +40,7 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Set up Emscripten latest
-        uses: mymindstorm/setup-emsdk@v12
+        uses: mymindstorm/setup-emsdk@v14
         with:
           version: ${{env.EM_VERSION}}
           actions-cache-folder: ${{env.EM_CACHE_FOLDER}}
```

```suggestion
          actions-cache-folder: ${{env.EM_CACHE_FOLDER}}
          cache-key: ${{ matrix.cache-name }}-emsdk
```

Something like this, perhaps?